kod/object/item.kod

```diff
@@ -1583,6 +1593,66 @@ messages:
          oItemAtt = Send(SYS,@FindItemAttByNum,#num=iNum);
          if NOT Send(oItemAtt,@ItemDropOnDeath)
          {
+            if IsClass(oItemAtt,&ItemAttTranscendant)
+            {
+               Send(Send(SYS,@FindItemAttByNum,#num=IA_TRANSCENDANT),
```

Comment:
Already have the ItemAtt object here, don't need to make the call to FindItemAttByNum again.